visualization/package.json

```diff
@@ -115,11 +116,13 @@
 		"@typescript-eslint/eslint-plugin": "^5.44.0",
 		"@typescript-eslint/parser": "^5.44.0",
 		"bestzip": "^2.2.0",
+		"core-js": "^3.35.1",
 		"electron-packager": "^17.1.2",
 		"eslint": "^8.28.0",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-unicorn": "^47.0.0",
 		"eslint-plugin-unused-imports": "^2.0.0",
+		"fake-indexeddb": "^5.0.2",
```

Comment:
Just use v5.0.0. It will work with Node 16 https://github.com/dumbmatter/fakeIndexedDB/releases

Reply:
Sorry thats wrong. Use latest version 4.